.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -3,12 +3,30 @@ name: 🐞 Bug Report
 about: Report a bug in this repo
 ---
 
-<!--
-Please consider including the answers below when relevant:
+### Environment
 
-How do you use Sentry:
-Sentry SaaS (sentry.io) or self-hosted/on-premise (open-source, which version?)
+**Server Version:** SaaS _or_ self-hosted *VERSION HERE*
+**SDK Version:** _e.g._: JavaScript 5.11.1, .NET 1.2.0
 
-Which SDK and version?
-e.g: JavaScript 5.11.1, .NET 1.2.0
--->
+### Steps to Reproduce
+
+1. What
+2. you
+3. did.
+
+### Expected Result
+
+What you thought would happen.
+
+### Actual Result
+
+What actually happened. Maybe a screenshot/recording?
+
+### Logs
+
+What you saw along the way.
+
+For **self-hosted**, e.g.:
+
+- latest install logs: `ls -1 sentry_install_log-*.txt | tail -1 | xargs cat`
```

Comment:
This will not be very useful for the SDK repos, can we keep this template more generic? In case where logs are required we can override the template in the repo specifically

Reply:
On the flip side, for the `sentry` and `onpremise` repos the SDK version will not be useful. I think in the interest of scalability—ideally we have zero overrides to manage—I think we should strive to make these global templates generic enough for all cases, and then let people simply ignore/delete the content that doesn't apply to them.

Is there logging guidance specific to SDKs that we would want to prompt for?